code/__DEFINES/colors.dm

```diff
@@ -71,6 +71,7 @@
 
 #define COLOR_BROWN "#BA9F6D"
 #define COLOR_DARK_BROWN "#997C4F"
+#define COLOR_ANOTHER_BROWN "#a9734f"
```

Comment:
Is it possible to describe this in relation to COLOR_BROWN and COLOR_DARK_BROWN? Is it lighter than COLOR_BROWN? If so, COLOR_LIGHTER_BROWN. For reference, look at the other names.

Reply:
I actually did Google this for inspiration but couldn't find anything I thought fit. Closest I got was "strong orange(brown)/strong tangelo" which I just thought was confusing. I'll call it COLOR_ORANGE_BROWN.